src/os/mixin/cesiummixin.js

```diff
@@ -172,11 +172,10 @@ os.mixin.cesium.loadCesiumMixins = function() {
     image.src = url;
   };
 
-
   /**
    * @suppress {accessControls|duplicate}
    */
-  Cesium.loadImage.defaultCreateImage = Cesium.loadImage.createImage;
+  Cesium.Resource._Implementations.createImage = Cesium.loadImage.createImage;
```

Comment:
One minor additional request. Can you set the above function to a local var and remove the suppression in the docs? This code will otherwise break when `Cesium.loadImage` is removed. `Cesium.loadImage.createImage` is no longer used internally by Cesium and we don't use it either, so there is no longer a need to replace the function.